packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -572,19 +572,19 @@ export function createLodestarMetrics(
     // Gossip block
     gossipBlock: {
       elapsedTimeTillReceived: register.histogram({
-        name: "lodestar_gossip_block_elappsed_time_till_received",
-        help: "Time elappsed between block slot time and the time block received via gossip",
+        name: "lodestar_gossip_block_elapsed_time_till_received",
```

Comment:
changing metric names is potentially problematic, needs to be confirmed by someone else that this is not a problem as long as the corresponding metric is also updated in the Grafana dashboard